tests/test_gpad_parser.py

```diff
@@ -112,3 +117,7 @@ def test_parse_2_0():
     vals[0] = "WB:WBGene00001189"
     result = to_association(list(vals), report=report, version=version)
     assert result.associations[0].subject.id == Curie("WB", "WBGene00001189")
+
+    # Test annotation property retrieval
+    contributors = result.associations[0].annotation_property_values(property_key="contributor-id")
```

Comment:
Same suggestion here as in `test_duplicate_key_annot_properties`

github is also complaining of lack of new line at the end of this file?